src/api/server.tsx

```diff
@@ -27,6 +68,32 @@ export const updateSettings = (config: LxdConfigPair): Promise<void> => {
   });
 };
 
+export const updateClusteredSettings = (
+  config: ClusterSpecificValues,
+  configName: string,
+): Promise<void> => {
+  return new Promise((resolve, reject) => {
+    Promise.allSettled(
+      Object.keys(config).map((memberName) => {
+        const memberNetwork = {
```

Comment:
```suggestion
        const config = {
```
Better name? I'd rename the incoming parameter in the root of the function `clusterValues` and this one will be the `config`.